ADR/0027-availability-probe-framework.md

```diff
@@ -0,0 +1,102 @@
+# 27. Availability Probe Framework
+
+* Date 2023-07-06
+
+## Status
+
+Accepted
+
+## Context
+
+As a product team, we want a composite system availability metric for both Production
+and Staging environments, so that we can make roadmap decisions with easy-to-understand
+feedback about the reliability of the system in mind.
+  
+This metric should tell us the overall up time of RHTAP with respect to services under
+the control of Red Hat.
```

Comment:
I don't think we need to word things in relation to being under control by a single entity. Can you generalize this?